engine/script/src/script_sys.cpp

```diff
@@ -70,6 +71,26 @@ union SaveLoadBuffer
      * @namespace sys
      */
 
+    char* Sys_SetupTableSerializationBuffer(int required_size)
```

Comment:
Use the 512kb fixed buffer for "small" files and a dynamically allocated buffer for large files